client/app/vps/modal/shortcut/reverse-dns/vps-reverse-dns.html

```diff
@@ -1,33 +1,31 @@
-<cui-modal>
-    <cui-modal-header data-on-dismiss="$ctrl.cancel()"></cui-modal-header>
-    <cui-modal-body data-title=":: 'vps_configuration_reversedns_title' | translate">
-        <div class="text-center" data-ng-if="$ctrl.loader.init || $ctrl.loader.save">
-            <oui-spinner></oui-spinner>
-        </div>
-        <div data-ng-if="!$ctrl.loader.init && !$ctrl.loader.save">
-            <label class="oui-label" data-translate="vps_configuration_reversedns_step1_question"></label>
-            <div class="oui-select">
-                <select class="oui-select__input"
-                        data-ng-options="ip as ip.ipAddress for ip in $ctrl.ips"
-                        data-ng-model="$ctrl.model.value">
-                    <option value="" data-ng-disabled="true" selected data-translate="vps_configuration_reversedns_step1_ip"></option>
-                </select>
-                <i class="oui-icon oui-icon-chevron-down" aria-hidden="true"></i>
-            </div>
-
+<form novalidate name="form" data-ng-submit="form.$valid && $ctrl.confirm()">
+    <oui-modal
+        data-title="{{:: 'vps_configuration_reversedns_title' | translate}}"
+        data-primary-label="{{ ::'common_confirm' | translate }}"
+        data-secondary-action="$ctrl.cancel()"
+        data-secondary-label="{{ ::'common_cancel' | translate }}"
+        data-on-dismiss="$ctrl.cancel()"
+        data-loading="$ctrl.loader.init || $ctrl.loader.save">
 
-            <div data-ng-if="$ctrl.model.value.ipAddress">
-                <label for="reverse" class="oui-label" data-translate="vps_configuration_reversedns_step2_newreverse"></label>
+        <oui-field data-label="{{ ::'vps_configuration_reversedns_step1_question' | translate }}">
+            <oui-select name="select-ip" 
+                model="$ctrl.model.value" 
+                data-title="{{::'vps_configuration_reversedns_step1_ip' | translate}}" 
+                placeholder="{{::'vps_configuration_reversedns_step1_ip' | translate}}" 
```

Comment:
`data-`

Reply:
@FredericEspiau you meaning `data-translate-attr="{ "placeholder": "vps_configuration_reversedns_step1_ip" }` right?